main.go

```diff
@@ -111,6 +112,30 @@ func handleSignals(fCache *filecache.FileCache, mList *memberlist.Memberlist) {
 	os.Exit(130)                // Ctrl-C received or equivalent
 }
 
+// configureNewRelic sets up and starts a Gorelic agent if we have a
+// New Relic license available.
+func configureNewRelic() *gorelic.Agent {
+	nrLicense := os.Getenv("NEW_RELIC_LICENSE_KEY")
+	var agent *gorelic.Agent
+	if nrLicense == "" {
+		log.Info("No New Relic license found, not starting an agent")
+
+		return nil
+	}
+
+	agent = gorelic.NewAgent()
+	svcName := os.Getenv("SERVICE_NAME")
+	envName := os.Getenv("ENVIRONMENT_NAME")
+	if svcName != "" && envName != "" {
+		agent.NewrelicName = fmt.Sprintf("%s-%s", svcName, envName)
+	}
+	agent.Verbose = true
```

Comment:
Stuff might get noisy in prod... Maybe have this enabled through an env var.

Reply:
I want to see what it does in dev, and will turn it off if it's noisy. We don't need a setting for that as I can't see why we'd need to flip it on/off  99% of the time.